pkg/extensions/search/schema.resolvers.go

```diff
@@ -419,7 +434,118 @@ func (r *queryResolver) GlobalSearch(ctx context.Context, query string) (*gql_ge
 	}, nil
 }
 
+// DependencyListForImage is the resolver for the DependencyListForImage field.
+func (r *queryResolver) DependencyListForImage(ctx context.Context, image string) ([]*gql_generated.ImageSummary, error) {
+	defaultStore := r.storeController.DefaultStore
+	imageList := make([]*gql_generated.ImageSummary, 0)
+
+	repoList, err := defaultStore.GetRepositories()
```

Comment:
We need to look into all stores. I think we have this problem in other queries as well.
We should have a new method in layoutUtils which returns all repos in all stores.